StripeFinancialConnections/StripeFinancialConnections/Source/Native/AccountPicker/AccountPickerFooterView.swift

```diff
@@ -111,3 +99,25 @@ final class AccountPickerFooterView: UIView {
         linkAccountsButton.isLoading = true
     }
 }
+
+private func CreateDataAccessLabel(
```

Comment:
Pattern inspired by @kgaidis-stripe ❤️

Reply:
2 years later, I still feel weird doing it 😅 but the thing that calms me down is that it makes sense if the goal is simplicity 🤷  (static function avoids state vs. instance functions; and the use of UIStackView avoids the need for UIView subclasses w/ autolayout - can just "compose" many static functions together)

In SwiftUI this is easy because doing this is basically the built-in standard